aspnetcore/tutorials/first-mvc-app/new-field.md

```diff
@@ -63,7 +63,7 @@ Update the `SeedData` class so that it provides a value for the new column. A sa
 [!code-csharp[Main](./start-mvc/sample2/src/MvcMovie/Models/SeedDataRating.cs?highlight=6&range=25-32)]
 
 >[!WARNING]
```

Comment:
@Rick-Anderson Are you saying to delete the code? Without it the reader won't know what to update.